src/søknad/steg/4-barnasbosted/BarnetsBostedEndre.tsx

```diff
@@ -122,43 +94,13 @@ const BarnetsBostedEndre: React.FC<Props> = ({
     fødselsdato,
     ident,
   } = forelder;
-  const jegKanIkkeOppgiLabel = hentTekst(
-    'barnasbosted.kanikkeoppgiforelder',
-    intl
-  );
 
   const erIdentUtfyltOgGylding = (ident?: string): boolean =>
     !!ident && erGyldigFødselsnummer(ident);
   const erFødselsdatoUtfyltOgGyldigEllerTomtFelt = (fødselsdato?: string) =>
     erGyldigDato(fødselsdato) || fødselsdato === '';
   const harForelderFraPdl = barn?.medforelder?.verdi?.navn || false;
 
-  useEffect(() => {
-    settForelder({
-      ...forelder,
-      kanIkkeOppgiAnnenForelderFar: {
-        label: jegKanIkkeOppgiLabel,
-        verdi: forelder.kanIkkeOppgiAnnenForelderFar?.verdi || false,
-      },
-    });
-    //eslint-disable-next-line
-  }, []);
-
-  useEffect(() => {
-    if (sisteBarnUtfylt === true && !erForelderUtfylt(forelder)) {
-      const nyBarneListe = lagOppdatertBarneliste(barneListe, barn, forelder);
-      settBarneListe(nyBarneListe);
-      settSisteBarnUtfylt(false);
-    }
-  }, [
-    sisteBarnUtfylt,
-    settSisteBarnUtfylt,
-    forelder,
-    barneListe,
-    barn,
-    settBarneListe,
-  ]);
```

Comment:
Logikk for å styre `sisteBarnUtfylt` er flyttet lenger ut for å slippe forvirrende `useEffect`